### ISSUE TITLE
evaluate these models

### ISSUE THREAD

**Issue by lixianyan2023** [rbiswasfc/eedi-mining-misconceptions#5]
Thank you for your answer, I have learned a lot. I am a beginner in large models, and step by step, I have learned the charm of large models through your code and ideas. I want to ask you another question: How can I evaluate the performance of the trained Retrievers, Reasoners, Re-rankers (Pointwise), and Re-rankers (Listwise) models? How can I obtain a metric to evaluate these models?

**Comment by lixianyan2023** [comment]
I remember you mentioned real-time tracking and evaluating the training process and data to improve the strategy. Could you please share some approaches for this？

**Comment by lixianyan2023** [comment]
I would like to ask:what is the training code for this? eedi-mining-misconceptions/conf/conf_ranker_prep.yaml..
Meanwhile,I found the 'teacher scores' inside the 'eedi-ranker-silver-v3-teacher' folder. How did you obtain these scores?" 
How is the 'silver_v3_qwen_14b.parquet' obtained in the 'conf_ranker_prep.yaml' configuration file?

**Comment by lixianyan2023** [comment]
Thank you for your answer. How do you get eedi-tutor-mix-v8? I mentioned that there are few teacher_logits and cot.
